examples/ConsumeWithRabbitMQAndDeadLetterExchange/ConsumeWithRabbitMQAndDeadLetterExchange.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I would also mention this new example in the README.md to make users aware of this functionality:
https://github.com/GDATASoftwareAG/motornet#examples